clippy_lints/src/modulo_arithmetic.rs

```diff
@@ -0,0 +1,106 @@
+use crate::consts::{constant, Constant};
+use crate::utils::{sext, span_note_and_lint};
+use if_chain::if_chain;
+use rustc::declare_lint_pass;
+use rustc::hir::*;
+use rustc::lint::{LateContext, LateLintPass, LintArray, LintPass};
+use rustc::ty::{self};
+use rustc_session::declare_tool_lint;
+
+declare_clippy_lint! {
+    /// **What it does:** Checks for modulo arithemtic.
+    ///
+    /// **Why is this bad?** The results of moudlo (%) operation migth differ
```

Comment:
Hey you have some typos here: `modulo` and `might`.

Reply:
Good catch, thanks! Fixed in https://github.com/rust-lang/rust-clippy/pull/4867/commits/6084c539e3d7690cfaf424206f552113249459c2